tests/integration/Cron/Workers/BeamerTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace MailPoet\Test\Cron\Workers;
+
+use Carbon\Carbon;
+use Codeception\Stub;
+use MailPoet\Cron\Workers\Beamer;
+use MailPoet\Settings\SettingsController;
+use MailPoet\WP\Functions as WPFunctions;
+
+class BeamerTest extends \MailPoetTest {
```

Comment:
First of all, I thought our coding standards require an empty line between the class header and the first method definition. The same on the end. I am surprised no check caught it. Could you please add those lines?